tests/foreman/installer/test_installer.py

```diff
@@ -1586,3 +1586,50 @@ def test_installer_timestamp_logs():
 
     :CaseAutomation: NotAutomated
     """
+
+
+def test_installer_pub_directory_accessibility(destructive_sat):
+    """Verify the public directory accessibility after disabling it from the custom-hiera
+
+    :id: 2ef78840-098c-4be2-a9e5-db60f16bb803
+
+    :steps:
+        1. Add the foreman_proxy_content::pub_dir::pub_dir_options:"+FollowSymLinks -Indexes"
+            in custom-hiera.yaml file.
+        2. Run the satellite-installer.
+        3. Access the https://SATELLITE_SERVER_NAME/pub
+
+    :expectedresults: After the custom_hiera.yaml settings, the public directory should not be
+       accessible.
+
+    :CaseImportance: High
+
+    :CaseLevel: System
+
+    :BZ: 1960801
+
+    :customerscenario: true
+    """
+    custom_hiera_settings = (
+        'foreman_proxy_content::pub_dir::pub_dir_options: ' '"+FollowSymLinks -Indexes"'
```

Comment:
Remnant of black merging two lines of strings
```suggestion
        'foreman_proxy_content::pub_dir::pub_dir_options: "+FollowSymLinks -Indexes"'
```